Tests/Shared.Specs/ExtensibilityRelatedEquivalencySpecs.cs

```diff
@@ -553,7 +553,7 @@ public void When_an_assertion_rule_is_added_it_appear_in_the_exception_message()
         }
 
         [Fact]
-        public void When_multiple_asertion_rules_are_added_they_should_be_evaluated_last_to_first()
+        public void When_multiple_assertion_rules_are_added_they_should_be_evaluated_last_to_first()
```

Comment:
Depending on how strict/isolated you want to be on commits, this should be moved to it's own commit.